Flutter/datagrid/column-types.md

```diff
@@ -556,6 +556,59 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid checkbox shape](images/column-types/flutter-datagrid-checkbox-shape.png)
 
+### Callbacks
+
+The `SfDataGrid` provides the following callback for checkbox interactions:
+
+* [onCheckboxValueChanged]() : This callback is triggered when the checkbox is selected or unselected (i.e., when the value of the checkbox changes).
+
```

Comment:
Explain the properties of the callback like the below reference,

https://help.syncfusion.com/flutter/datagrid/swiping#swipe-callbacks